roles/nvidia_gpu_exporter/tasks/main.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Preflight
+  ansible.builtin.include_tasks:
+    file: preflight.yml
+  tags:
+    - nvidia_gpu_exporter_install
+    - nvidia_gpu_exporter_configure
+    - nvidia_gpu_exporter_run
+
+- name: Install
+  ansible.builtin.include_role:
+    name: prometheus.prometheus._common
+    tasks_from: install.yml
+  vars:
+    _common_local_cache_path: "{{ nvidia_gpu_exporter_local_cache_path }}"
+    _common_binaries: "{{ _nvidia_gpu_exporter_binaries }}"
+    _common_binary_install_dir: "{{ nvidia_gpu_exporter_binary_install_dir }}"
+    _common_binary_url: "{{ nvidia_gpu_exporter_binary_url }}"
+    _common_checksums_url: "{{ nvidia_gpu_exporter_checksums_url }}"
+    _common_system_group: "{{ nvidia_gpu_exporter_system_group }}"
+    _common_system_user: "{{ nvidia_gpu_exporter_system_user }}"
+    _common_binary_unarchive_opts: ['--strip-components=1']
+  tags:
+    - nvidia_gpu_exporter_install
+
+- name: SELinux
+  ansible.builtin.include_role:
+    name: prometheus.prometheus._common
+    tasks_from: selinux.yml
+  vars:
+    _common_selinux_port: "{{ nvidia_gpu_exporter_web_listen_address | urlsplit('port') }}"
+  when: ansible_selinux.status == "enabled"
+  tags:
+    - nvidia_gpu_exporter_configure
+
+- name: Configure
+  ansible.builtin.include_tasks:
+    file: configure.yml
+  tags:
+    - nvidia_gpu_exporter_configure
+
+- name: Ensure Nvidia GPU Exporter is enabled on boot
+  become: true
+  ansible.builtin.systemd:
+    daemon_reload: true
+    name: nvidia_gpu_exporter
+    enabled: true
+    state: started
+  when:
+    - not ansible_check_mode
+  tags:
+    - nvidia_gpu_exporter_run
```

Comment:
```suggestion
  tags:
    - nvidia_gpu_exporter
    - run
    - nvidia_gpu_exporter_run
```

Reply:
Same as last reply.
The tags are referred to node_exporter's.